packages/espresso/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.3.1
+
+* Updates espresso version to 3.6.1
```

Comment:
Maybe call this `androidx.test.espresso` instead of just `espresso` to make it less confusing, since `espresso` is also the package name?